Doc/whatsnew/3.7.rst

```diff
@@ -865,143 +1667,245 @@ argument to generate a compressed archive.  A command line option
+:issue:`29452`, and :issue:`29286`.)
+
+Various optimizations have reduced Python startup time by 5% on Linux and
+up to 30% on macOS.
+(Contributed by Victor Stinner, INADA Naoki in :issue:`29585`.)
+
+Method calls are now up to 20% faster due to the bytecode changes which
+avoid creating bound method instances.
+(Contributed by Yury Selivanov and INADA Naoki in :issue:`26110`.)
+
+The :mod:`asyncio` received a number of notable optimizations for commonly
+used functions.
+See :ref:`optimizations in asyncio <whatsnew37-asyncio-optimizations>` for
+details.
+
+:func:`sorted` and :meth:`list.sort` have been optimized for common cases
+to be up to 40-75% faster.
+(Contributed by Elliot Gorokhovsky in :issue:`28685`.)
+
+:meth:`dict.copy` is now up to 5.5 times faster.
+(Contributed by Yury Selivanov in :issue:`31179`.)
+
+:func:`hasattr` and :func:`getattr`` are now about 4 times faster when
+*name* is not found and *obj* does not override :meth:`object.__getattr__`
+or :meth:`object.__getattribute__`.
+(Contributed by INADA Naoki in :issue:`32544`.)
+
+Searching for certain Unicode characters (like Ukrainian capital "Є")
+in a string was up to 25 times slower than searching for other characters.
+It is now only 3 times slower in the worst case.
+(Contributed by Serhiy Storchaka in :issue:`24821`.)
+
+The :func:`collections.namedtuple` factory has been reimplemented to
+make the creation of named tuples 4 to 6 times faster.
+(Contributed by Jelle Zijlstra with further improvements by INADA Naoki,
+Serhiy Storchaka, and Raymond Hettinger in :issue:`28638`.)
+
+:meth:`date.fromordinal` and :meth:`date.fromtimestamp` are now up to
+30% faster in the common case.
+(Contributed by Paul Ganssle in :issue:`32403`.)
+
+Fast implementation from standard C library is now used for the
+:func:`math.erf` and :func:`math.erfc` functions.
+(Contributed by Serhiy Storchaka in :issue:`26121`.)
+
+The :func:`os.fwalk` function is now up to 2 times faster thanks to
+the use of :func:`os.scandir`.
+(Contributed by Serhiy Storchaka in :issue:`25996`.)
+
+The speed of the :func:`shutil.rmtree` function has been improved by
+20--40% thanks to the use of the :func:`os.scandir` function.
+(Contributed by Serhiy Storchaka in :issue:`28564`.)
+
+Optimized case-insensitive matching and searching of :mod:`regular
+expressions <re>`.  Searching some patterns can now be up to 20 times faster.
+(Contributed by Serhiy Storchaka in :issue:`30285`.)
+
+:func:`re.compile` now converts ``flags`` parameter to int object if
+it is ``RegexFlag``.  It is now as fast as Python 3.5, and faster than
+Python 3.6 by about 10% depending on the pattern.
+(Contributed by INADA Naoki in :issue:`31671`.)
+
+The :meth:`~selectors.BaseSelector.modify` methods of classes
+:class:`selectors.EpollSelector`, :class:`selectors.PollSelector`
+and :class:`selectors.DevpollSelector` may be around 10% faster under
+heavy loads.  (Contributed by Giampaolo Rodola' in :issue:`30014`)
```

Comment:
use correct accent in contributor's name instead of trailing apostrophe?

Reply:
This is how Giampaolo is spelling his name.